packages/backend/helpers/flattenObj.ts

```diff
@@ -0,0 +1,11 @@
+export const flattenObj = (obj: any, exceptionList: any[] = [], parent?: any, res: any = {}) => {
+    for (let key in obj) {
```

Comment:
@Hvinder Is there a good reason for this to be a method to be written by us? I believe we are using lodash which has this function.

Reply:
I needed the function to take an additional argument of exception list (to not flatten special objects like 'additional' & 'associations'). The actual flatten logic might still be done using lodash, but I found writing this easier (ref - https://stackoverflow.com/questions/42645376/flatten-nested-object-using-lodash)